edx_sga/sga.py

```diff
@@ -504,25 +593,45 @@ def remove_grade(self, request, suffix=''):
         return Response(json_body=self.staff_grading_data())
 
     def is_course_staff(self):
+        """
+         Check if user is course staff
+        """
         return getattr(self.xmodule_runtime, 'user_is_staff', False)
 
     def is_instructor(self):
+        """
+        Check if user role is instructor
+        """
         return self.xmodule_runtime.get_user_role() == 'instructor'
 
     def show_staff_grading_interface(self):
+        """
+        show staff grading interface
+        """
         in_studio_preview = self.scope_ids.user_id is None
         return self.is_course_staff() and not in_studio_preview
 
     def past_due(self):
+        """
+        past due
+        """
         due = get_extended_due_date(self)
         if due is not None:
             return _now() > due
         return False
 
     def upload_allowed(self):
+        """
+        upload allowed
```

Comment:
Can you change it to something like "The upload is allowed if due date has not passed and no score has yet been assigned."